compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -53,6 +53,9 @@ class CircleTensoInfo
   const ShapeDescription &shape(void) const { return _shape; }
   void shape(const ShapeDescription &shape) { _shape = shape; }
 
+  void no_shape(void) { _no_shape = true; }
```

Comment:
I found `const_node->no_shape(true);` at below code @.@
Is it different with this function?

Reply:
`node->no_shape(true);` is for `CircleNode` to set/reset. Reset(`no_shape(false);`) is not used yet but as an IR, we may use in some transformations.
Here, I only needed `no_shape()` as set :)